closure/goog/editor/field.js

```diff
@@ -2799,4 +2801,4 @@ goog.editor.Field.prototype.getIframeAttributes = function() {
   }
 
   return {'frameBorder': 0, 'style': iframeStyle};
-};
+};
```

Comment:
I think I hallucinated removing two spaces there, based on the Travis-clang report.

Will do the changes required asap.